### PR TITLE
allow env vars in docker entrypoint

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,5 @@
+ADD: allow usage of env vas with in docker entry point
```
Comment (fgalan) [inline ChangeLog:1]:
```suggestion
ADD: allow usage of env var in docker entry point
```

(Unsure)

File: DOCKER.md [hunk 2]
```diff
@@ -48,6 +48,19 @@ By default wait until mysql database is operative at maximum of default DBTIMEOU
     - DBTIMEOUT=100
 ```
 
+
+
+Additionally, the following environment variables are available for keypass docker
+
+| Environment variable        | Configuration attribute   | Default value             |
+|:----------------------------|:--------------------------|:--------------------------|
+| DB_HOST_VALUE               |                           | localhost:3306            |
```
Comment (fgalan) [inline DOCKER.md:57]: Maybe the should use the KEYPASS_ prefix? E.g. KEYPASS_DB_HOST_VALUE

File: ChangeLog (hunk 1 above)
Comment (AlvaroVega) [reply ChangeLog:1]: Already fixed

File: DOCKER.md (hunk 2 above)
Comment (AlvaroVega) [reply DOCKER.md]: fixed in https://github.com/telefonicaid/fiware-keypass/pull/128/commits/a60881ad7ac4c77f7d820677e149acab30e50752

**Pull request merged**